lib/tests/streamlit/scriptrunner/ScriptRunner_test.py

```diff
@@ -286,7 +286,16 @@ def test_multiple_scriptrunners(self):
         for runner in runners:
             runner.start()
 
-        time.sleep(0.1)
+        t0 = time.time()
+
+        # Make this test a little less flaky, by waiting for all runners to
+        # run for long enough (measured in number of deltas produced).
+        while time.time() - t0 < 1:
+            time.sleep(0.1)
+            if all(len(runner.deltas()) >= 9 for runner in runners):
+                # widgets_script.py has 8 deltas, then a 1-delta loop. If 9
```

Comment:
Should we also add comment to `widgets_script` (so that we accidentally don't break this behavior)?